lib/json0.js

```diff
@@ -669,7 +669,7 @@ json.transformComponent = function(dest, c, otherC, type) {
     } else if (otherC.oi !== void 0) {
       if (c.oi !== void 0 && c.p[common] === otherC.p[common]) {
         // left wins if we try to insert at the same place
-        if (type === 'left') {
+        if (type === 'left' && !deepEqual(otherC.oi, c.oi)) {
```

Comment:
They can differ with `od` though no?

Reply:
Good prompt: I added a test case and realised that ops with no `od` are handled by different code to ops with an `od` 💀 

However, apart from the quirks of this codebase, I don't think it matters if they have different `od`: they should still no-op. Ultimately, `transform(a, b)` is saying: "make `a` compatible with a document where `b` has already been applied." So if we've already applied `b`, it's `oi` is the current value. Applying `a` — with the *same* `oi` — would result in no change, regardless of what its `od` is. Therefore, it should be a no-op.